### PR TITLE
Update Travis matrix

### PR DESCRIPTION
**Pull request by samhamilton**
Add Elixir 1.6 and 1.7 to test matrix

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
+    - elixir: 1.3
+      otp_release: 21.0
+    - elixir: 1.4
+      otp_release: 21.0
+    - elixir: 1.5
+      otp_release: 21.0
+    - elixir: 1.6
+      otp_release: 18.3
+    - elixir: 1.7
+      otp_release: 18.3
```
Comment (ericmj) [inline .travis.yml]: It looks easier to explicitly include versions rather than excluding.

**Comment by josevalim** [comment]
Hi @samhamilton, can you please explicitly include versions as @ericmj proposed? Thank you!

**Comment by samhamilton** [comment]
@josevalim @ericmj done - perhaps it's just me but this seems less clear now, i probably got used to the other way around, total lines is higher but if you guys are happy then that's all that matters!

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**